cli/command/load_balancer.go

```diff
@@ -84,6 +89,11 @@ func (l *LoadBalancerCommand) Command() cli.Command {
 						Value: dhc.UnhealthyThreshold,
 						Usage: "Number of consecutive failures required to count as unhealthy",
 					},
+					cli.StringFlag{
+						Name:  "type",
+						Value: string(config.DefaultLoadBalancerType),
```

Comment:
Why don't just have `config.DefaultLoadBalancerType` be a `const string`?

Reply:
LoadBalancerType is
```go
type LoadBalancerType string
```